vignettes/SEIRD_Model_Compartments.Rmd

```diff
@@ -0,0 +1,249 @@
+devtools::load_all()
+library(comomodels)
+library(knitr) # table layout 
+# library(plotly) # interactive plot
+# library(kableExtra) # plot and table arrangement
+```
+
+## Introduction
+This document explains the basics of the SEIaImIsRD model. This model is an extension of the SEIRD model where instead of having one population of infected individuals there are three, each with their own level of symptoms. In this model, populations of asymptomatic, mildly symptomatic and severely symptomatic infectious individuals are included. The motivation for including these separate populations of infectious individuals comes from the fact that individuals with severe symptoms are much more likely to be detected as infected than asymptomatic individuals, which affects how correct the count of cases is and perhaps how effective a quarantining campaign will be. Furthermore, the severeness of symptoms can correlate with the infectiousness of the individual, and their probablity of and time to recovery or disease-related death, thus affecting epidemic dynamics. 
+
+Here, we introduce the object class `SEIaImIsRD` included in the `comomodels` package, which lets the user simulate the SEIaImIsRD model, choosing the initial conditions and parameter values.
+
+## The SEIaImIsRD Model
+The SEIaImIsRD model describes the following population groups:
+
+* **Susceptible** ($S$) - individuals that have not been exposed to the virus, which can be infected by infectious individuals;
+
+* **Exposed** ($E$) - individuals that have been exposed to the virus and have been infected, but are not yet infectious to others as they are within the incubation period;
+
+* Infectious - individuals that can spread the virus to others. In this model, we subdivide Infectious individuals into three sub-categories depending on their symptoms.  
+  * **Asymptomatic infectious** ($I_{asymptomatic}$, or $I_a$ for short);
+  * **Mildly symptomatic infectious** ($I_{mild}$, or $I_m$ for short),
+  * **Severely symptomatic infectious** ($I_{severe}$, or $I_s$ for short).
+Each category is set with different transmission parameters.
+
+* **Recovered** ($R$) - individuals that have recovered from the disease and obtained immunity to it. They cannot be infected again until they lose their immunity and return to the susceptible population.
+
+
+The SEIaImIsRD model consists of an ordinary differential equation (ODE) system of six ODEs, each describing the rate of the size changing of a population group evolves over time:
+
+
+$$\frac{\text{d}S}{\text{d}t} = -S (\beta_{I_a}I_a + \beta_{I_m}I_m + \beta_{I_s}I_s) + \omega R,$$
+$$\frac{\text{d}E}{\text{d}t} = S (\beta_{I_a}I_a + \beta_{I_m}I_m + \beta_{I_s}I_s) - \kappa E,$$
+$$\frac{dI_{a}}{dt} = \eta_{E->I_a} \kappa E - (\gamma_{I_a} + \mu_{I_a})I_{a},$$
+$$\frac{dI_{m}}{dt} = \eta_{E->I_m} \kappa E - (\gamma_{I_m} + \mu_{I_a})I_{m},$$
+$$\frac{dI_{s}}{dt} = \eta_{E->I_s} \kappa E - (\gamma_{I_s} + \mu_{I_a})I_{s},$$
```

Comment:
These parameter names don't currently match those in the documentation. Can we make these the same across both?

Reply:
It's for simplification when writing a lot of them. I mentioned them here: 
```
  * **Asymptomatic infectious** ($I_{asymptomatic}$, or $I_a$ for short);
  * **Mildly symptomatic infectious** ($I_{mild}$, or $I_m$ for short),
  * **Severely symptomatic infectious** ($I_{severe}$, or $I_s$ for short).
```